### ISSUE TITLE
Installation for CPU tensor flow still trying to use CUDA

### ISSUE THREAD

**Issue by vigji** [DeepLabCut/DeepLabCut#327]
Running DeepLabCut 2.0.6.3 in an anaconda3 env created from the dlc-windowsCPU config file, on a Windows 10 desktop computer.

When I try to train the network in the Openfield tutorial, I get the following error from tensorflow:
`
InternalError: cudaGetDevice() failed. Status: CUDA driver version is insufficient for CUDA runtime version
`

I think this happens because by default tensor flow try to use CUDA even if I specify that I wanted to run DeepLabCut with CPU and not GPU.
To make the training working, I had to modify line 100 of pose_estimation_tensorflow/train.py:
`sess = tf.Session()`

With the following:
sess = tf.Session(config=tf.ConfigProto(device_count={'GPU': 0}))

To ensure that it does not try to use the GPU drivers that apparently are not usable (but I am no expert there).
I was wondering whether there is something that I am missing in the configuration of the training. I thought it would have been enough to setup the environment dlc-windowsCPU for that do be dealt with.

**Comment by MMathisLab** [comment]
There is a variable you pass that sets GPU to use to None (`gputouse=None`). Please see here:

https://github.com/AlexEMG/DeepLabCut/blob/master/docs/functionDetails.md#g-train-the-network

Also, please fill out the issue completely, i.e. which deeplabcut, 2.0.6.3, etc

**Comment by vigji** [comment]
Thank you very much!
It would be convenient to make it explicit somewhere in the notebook so that Windows users trying to test it can solve it before

**Comment by shaanchandra** [comment]
I had the same issue. I trained in CPU mode initially (Deeplabcut version > 2.0.7.0, Ubuntu 18.04, CUDA 10.0, tensorflow 1.14) and it ran out of the box.

Later I set up my CUDA environment and everything (Excellent docs for that BTW on GitHub and thank you for that !!) and tried training on GPU by using **gputouse = 0** but got memory error (realized I had a 2GB GPU and authors recommend minimum 8GB?). So I tried switching back to CPU with **gputouse = none** but it kept using GPU!!??!!

What @vigji  suggested here is what solved it for me eventually. Spent like 3 hours figuring why is it still using GPU (even printed the if condition that checks if GPU=None, and it used to pass, still the training would start in GPU and give memory errors).

But thank you!
